lib/api.js

```diff
@@ -577,6 +577,22 @@ api.declare({
 });
 
 /**
+ * List AMIs and their usage
+ */
+api.declare({
+  method: 'get',
+  route: '/internal/ami-usage',
+  name: 'amiUsage',
+  title: 'See the list of AMIs and their usage',
```

Comment:
nit: s/See/Show/

Reply:
I would go with "show" as well but other GET methods use "See the lists ..." so I decided to be consistent

Reply:
Let's stick with consistency.